templates/workflow/submissionIdentification.tpl

```diff
@@ -0,0 +1,29 @@
+{**
+ * templates/workflow/submissionIdentification.tpl
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * Show submission identification component
+ *}
+
+<span class="pkpWorkflow__identificationId">
+	{{ submission.id }}
+</span>
+<span class="pkpWorkflow__identificationDivider">
+	/
+</span>
+
+{if !!$currentPublication.authorsStringShort}
+	<span class="pkpWorkflow__identificationAuthor">
+		{{ currentPublication.authorsStringShort }}
+	</span>
+	<span class="pkpWorkflow__identificationDivider">
+		/
+	</span>
+{/if}
```

Comment:
This won't work, because the author string can change after the page is loaded (for example, when editing the contributors). You'll need to use `v-if` and tie it to the state, like you've done with the preview in the UI Library.

Reply:
You are right. Changed.